libcontainerd/client_windows.go

```diff
@@ -94,6 +94,7 @@ type containerInit struct {
 	MappedDirectories       []mappedDir // List of mapped directories (volumes/mounts)
 	SandboxPath             string      // Location of unmounted sandbox (used for Hyper-V containers)
 	HvPartition             bool        // True if it a Hyper-V Container
+	ContainerType           string      `json:",omitempty"` // Represents the enum value for the requested container type
```

Comment:
Nit. Seems a strange place in terms of ordering to put this. If you want to be nice, you could alphabetise the structure :smile: